### PR TITLE
Adding go server version in server info response body.

### PR DESCRIPTION
**Pull request by SanjanaB**

### CODE REVIEW

File: plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/common/serverinfo/MessageHandlerForServerInfoRequestProcessor1_0.java [hunk 1]
```diff
@@ -28,6 +30,7 @@ public String serverInfoToJSON(String serverId, String siteUrl, String secureSit
         object.addProperty("server_id", serverId);
         object.addProperty("site_url", siteUrl);
         object.addProperty("secure_site_url", secureSiteUrl);
+        object.addProperty("server_version", CurrentGoCDVersion.getInstance().fullVersion());
```
[the remark this answers is not in the document]
Comment (SanjanaB) [reply plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/common/serverinfo/MessageHandlerForServerInfoRequestProcessor1_0.java]: okay will change.

**Comment by SanjanaB** [comment]
Sure if it makes sense to add all version bits to the api.
Will add the following fields:
Full version
Dist version
Go Version

Anyother field is required?

**Comment by ketan** [comment]
I think the version API has all the bits that are needed to compose all
parts of full version string `x.y.z-PQR (long-git-sha)`

**Comment by SanjanaB** [comment]
Keeping this open till we decide how to handle Api versioning. cc @ketan

**Pull request closed without merging**